projects/step-core/src/lib/modules/dynamic-forms/components/dynamic-field-editor/dynamic-field-editor.component.ts

```diff
@@ -31,6 +31,17 @@ export class DynamicFieldEditorComponent implements OnChanges {
   handleChange(value?: DynamicFieldGroupValue): void {
     this.internalValue = value;
 
+    if (value) {
+      Object.keys(value)
+        .filter((key) => value[key] === 'string')
```

Comment:
```suggestion
        .filter((key) => typeof value[key] === 'string')
```
Pretty sure you forgot the typeof

Reply:
yeah thank you for noticing, I was refactoring the code before making a PR and I've missed it, cheers